csharp/ql/lib/semmle/code/csharp/Type.qll

```diff
@@ -138,6 +138,9 @@ class ValueOrRefType extends Type, Attributable, @value_or_ref_type {
   /** Gets an immediate subtype of this type, if any. */
   ValueOrRefType getASubType() { result.getABaseType() = this }
 
+  /** Gets an immediate supertype of this type, if any. */
+  ValueOrRefType getASupertype() { this.getABaseType() = result }
```

Comment:
It looks like the capitalisation between Java and C# is a bit different - so for C# we probably need to name this predicate `getASuperType` to streamline it with the other predicate names.